content/en/docs/creating-addons/testing/install-specific-version-of-addon.md

```diff
@@ -0,0 +1,191 @@
+       kind: Addon
+       metadata:
+         annotations:
+         ...   
+         creationTimestamp: "2023-03-20T19:07:08Z"
+         finalizers:
+         - addons.managed.openshift.io/cache
+         ...
+       spec:
+         displayName: Reference Addon
+         ...
+         pause: false
+         version: 0.6.7
+       status:
+         conditions:
+         - lastTransitionTime: "2023-03-20T19:08:10Z"
+           message: ""
+           observedGeneration: 2
+           reason: FullyReconciled
+           status: "True"
+           type: Available
+         - lastTransitionTime: "2023-03-20T19:08:10Z"
+           message: Addon has been successfully installed.
+           observedGeneration: 2
+           reason: AddonInstalled
+           status: "True"
+           type: Installed
+         lastObservedAvailableCSV: redhat-reference-addon/reference-addon.v0.6.7
+         observedGeneration: 2
+         observedVersion: 0.6.7
+         phase: Ready
+         ```
+         In this example, you can see the add-on version is set to `0.6.7` and `AddonInstalled` status is `True`.
+
+5. If you do not want the add-on service to automatically upgrade to the latest version after installation, delete the add-on upgrade policy after installation is complete.
+
+    1. List the upgrade policies:
+
+        **Example**
+        ```bash
+        $ ocm get /api/clusters_mgmt/v1/clusters/$CLUSTER_ID/addon_upgrade_policies
+        {
+       "kind": "AddonUpgradePolicyList",
+       "page": 1,
+       "size": 1,
+       "total": 1,
+       "items": [
+         {
+          "kind": "AddonUpgradePolicy",
+          "id": "991a69a5-ce33-11ed-9dda-0a580a8308f5",
+          "href": "/api/clusters_mgmt/v1/clusters/22ogsfo8kd36bk280b6bqbi7l03micmm/addon_upgrade_policies/991a69a5-ce33-11ed-9dda-0a580a8308f5",
+          "schedule": "0,15,30,45 * * * *",
+          "schedule_type": "automatic",
+          "upgrade_type": "ADDON",
+          "version": "",
+          "next_run": "2023-03-29T19:30:00Z",
+          "cluster_id": "22ogsfo8kd36bk280b6bqbi7l03micmm",
+          "addon_id": "reference-addon"
+         }
+        ]
+        }
+        ```
+
+    2. Delete the add-on upgrade policy:
+
+        **Syntax**
+        ```
+        ocm delete /api/clusters_mgmt/v1/clusters/$CLUSTER_ID/addon_upgrade_policies/<addon_upgrade_policy_id>
+        ```
+        **Example**
+        ```bash
+        $ ocm delete /api/clusters_mgmt/v1/clusters/$CLUSTER_ID/addon_upgrade_policies/991a69a5-ce33-11ed-9dda-0a580a8308f5     
+        ```
+    3. Verify the upgrade policy no longer exists:
+
+        **Syntax**
+        ```
+        ocm get /api/clusters_mgmt/v1/clusters/$CLUSTER_ID/addon_upgrade_policies | grep <addon_upgrade_policy_id>
+        ```
+
+        **Example**
+        ```bash
+        $ ocm get /api/clusters_mgmt/v1/clusters/$CLUSTER_ID/addon_upgrade_policies | grep 991a69a5-ce33-11ed-9dda-0a580a8308f5
+        ```
```

Comment:
Now that we removed the automated policy, I'm missing the steps to actually upgrade the addon when needed... (it's by creating a manual policy, but I don't know the steps by heart)

Reply:
I found an example, just to give some pointers...

```
 echo "{
         \"upgrade_type\": \"ADDON\",
         \"cluster_id\": \"${CLUSTER_ID}\"
         \"addon_id\": \"${addon_id}\"
         \"version\": \"${addon_version}\",
         \"schedule_type\": \"manual\",
     }" | ocm post "${API_CLUSTERS}/${CLUSTER_ID}/addon_upgrade_policies"
```

Reply:
It needs some testing though 😌

Reply:
Thanks! I need to test this yet, but my cluster wasn't cooperating today!

Reply:
@maryfrances01 Has testing been completed? Or did issues pop up that we should discuss with Service Development?